tools/src/dwarfs_main.cpp

```diff
@@ -1245,7 +1246,9 @@ void usage(std::ostream& os, std::filesystem::path const& progname) {
      << "USING HIGH-LEVEL FUSE API\n\n"
 #endif
      << "Usage: " << progname.filename().string()
-     << " <image> <mountpoint> [options]\n\n"
+     << " <image> <mountpoint> [options]\n"
+     << "       " << progname.filename().string()
+     << " <image> --automount  [options]\n\n"
```

Comment:
A few things:

- I wonder if the name `automount` is misleading. IIRC, traditionally, `automount` means "automatically mount if the mount point is accessed", but that's not quite what this does. What about `--auto-mountpoint`?
- The whole code should be guarded by `#ifndef _WIN32`; on Windows, the mount point mustn't actually exist and will be created/removed automatically, since that's how WinFsp/Windows works.